.env.example

```diff
@@ -14,8 +14,7 @@ KODI_PORT=
 KODI_USERNAME=
 KODI_PASSWORD=
 
-SKILL_APPID=
-SKILL_VERIFY_CERT=
```

Comment:
Oh ok.  So we probably want the cert verification enabled in Flask-Ask at all times anyway.